CMakeLists.txt

```diff
@@ -328,7 +328,11 @@ if((CLANG_ASAN_UBSAN OR CLANG_MSAN OR CLANG_TSAN) AND NOT CMAKE_C_COMPILER_ID MA
   message(FATAL_ERROR "Sanitizers are only supported for Clang.")
 endif()
 
-option(ENABLE_JEMALLOC "enable jemalloc" ON)
+if(CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
+  option(ENABLE_JEMALLOC "enable jemalloc" OFF)
+else()
+  option(ENABLE_JEMALLOC "enable jemalloc" ON)
+endif()
```

Comment:
I see now that `option(... OFF)` only sets the default value, doesn't override a provided value--my mistake.

Though if someone wants to fix the issue, they could modify the CMakeLists.txt. In the meantime it's nice to have a `message()` for visibility.